src/devtools/client/debugger/src/components/Editor/Breakpoints/Panel/PanelSummary/index.tsx

```diff
@@ -106,8 +102,23 @@ function PanelSummary({
   return (
     <div className="summary space-x-2" onClick={e => handleClick(e, "logValue")}>
       <div className="options items-center flex-col flex-grow">
-        <Condition {...{ isEditable, handleClick, conditionValue }} />
-        <Log {...{ isEditable, handleClick, logValue }} hasCondition={!!conditionValue} />
+        {conditionValue ? (
+          <Condition
+            {...{
```

Comment:
We should either user an object spread or JSX attributes. This is a code style thing but we should probably land on a default. _My preference_ is to use JSX attributes unless we're spreading an existing object.

```
  // uses both but doesn't construct an object just to spread it
  <Condition {...rest} a={a} b={b} />
```